buildSrc/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -57,20 +57,25 @@ class FladlePluginDelegate {
   }
 
   private fun TaskContainer.createTasksForConfig(base: FlankGradleExtension, config: FladleConfig, project: Project, name: String) {
+    val writeConfigProps = register("writeConfigProps$name", YamlConfigWriterTask::class.java, base)
```

Comment:
@CristianGM curious, why did you remove passing the `config` in the constructor here and instead set it during the configuration?

Reply:
can I declare a Task input as a constructor parameter?

Reply:
I guess I could just add a getter to declare the input.
But actually is that I never used constructor parameters for a task (unless for injected services) and the documentation doesn't mention them a lot.

Reply:
Yeah, like this: https://github.com/runningcode/fladle/pull/162/files#diff-0a17c2db8a8d3d5a093b29f3284b93b5R16